babel/dates.py

```diff
@@ -694,7 +718,7 @@ def format_time(time=None, format='medium', tzinfo=None, locale=LC_TIME):
     return parse_pattern(format).apply(time, locale)
 
 
-def format_skeleton(skeleton, datetime=None, tzinfo=None, locale=LC_TIME):
+def format_skeleton(skeleton, datetime=None, tzinfo=None, fuzzy=True, locale=LC_TIME):
```

Comment:
More tests needed, e.g., with `fuzzy == False` and `tzinfo != None`.  Use the `tests` package if you don't want to pollute the *docstring*.